react/components/Onboarding/components/UI/CTAButton/index.js

```diff
@@ -0,0 +1,13 @@
+import { space } from 'styled-system';
+import { preset } from 'react/styles/functions';
+import styled from 'styled-components';
+import GenericButton from 'react/components/UI/GenericButton';
+
+const CTAButton = styled(GenericButton).attrs({
+  f: 5,
+})`
+  ${space}
+  ${preset(space, { mt: 6 })}
```

Comment:
You don't need to include space, nor set the preset—GenericButton implements space so you could just set mt in the attrs.

Reply:
👍